include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,563 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
+{
+    // ! In Vulkan: These will be reported based on availability of an extension and will be enabled by enabling an extension
+    // Table 51. Extension Feature Aliases (vkspec 1.3.211)
+    // Extension                               Feature(s)
+    // VK_KHR_shader_draw_parameters           shaderDrawParameters
+    // VK_KHR_draw_indirect_count              drawIndirectCount
+    // VK_KHR_sampler_mirror_clamp_to_edge     samplerMirrorClampToEdge
+    // VK_EXT_descriptor_indexing              descriptorIndexing
+    // VK_EXT_sampler_filter_minmax            samplerFilterMinmax
+    // VK_EXT_shader_viewport_index_layer      shaderOutputViewportIndex, shaderOutputLayer
```

Comment:
isn't this a comment that should go into `CVulkan` file?

Reply:
sure, good idea